README.md

```diff
@@ -14,15 +15,17 @@
 First, install the library from npm with the package manager of your choice:
 
 ```sh
-yarn add @expensify/react-native-live-markdown
-npm install @expensify/react-native-live-markdown --save
-npx expo install @expensify/react-native-live-markdown
+yarn add @expensify/react-native-live-markdown react-native-reanimated expensify-common
```

Comment:
Is `expensify-common` needed when using a custom parser? If so, shouldn't it be a peer dependency?